### ISSUE TITLE
0.4.0-highlight.0 release messes with caret range version resolution

### ISSUE THREAD

**Issue by taion** [system-ui/theme-ui#1006]
**Describe the bug**
By lexicographic sorting, `^` and `>=` ranges on the `0.4.0` alphas resolves to `0.4.0-highlight.0` rather than the more recent (and likely intended) `0.4.0-alpha.3`. This happens both for external things that depend on the alphas, and internally within the monorepo as well.

**To Reproduce**
Run `yarn add gatsby`.

**Expected behavior**
These ranges resolve to `0.4.0-alpha.3`, the actual latest alpha.

**Screenshots**
From my `yarn.lock`:

```
"theme-ui@>= 0.4.0-alpha.0", theme-ui@^0.4.0-alpha.3:
  version "0.4.0-highlight.0"
```

```
"@theme-ui/css@>= 0.4.0-alpha.0", "@theme-ui/css@^0.4.0-highlight.0":
  version "0.4.0-highlight.0"
```

**Additional context**
I noticed this because `0.4.0-highlight.0` exports some really odd types under `@theme-ui/css` that make building unrelated things fail due to `CSSSelectorObject` and `VariantProperty` being private, which has since been fixed in alpha.3.

**Comment by atanasster** [comment]
Encountering the same issue

**Comment by taion** [comment]
Can we do `0.4.1-alpha.0`? Or else bump the prerelease tag to `rc`.

**Comment by jxnblk** [comment]
Packages should now be published as `0.4.0-rc.1` with the `@next` tag. Please feel free to reopen this issue if the problem persists or you run into any other issues
